src/main/java/org/launchcode/liftoffrecipetracker/models/Recipe.java

```diff
@@ -0,0 +1,107 @@
+package org.launchcode.liftoffrecipetracker.models;
+
+import javax.persistence.Entity;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+import java.util.HashMap;
+
+
+@Entity
+public class Recipe extends AbstractEntity {
+	//properties
+	private HashMap<String, String> ingredients;
+
+	@NotBlank(message = "Recipe name required.")
+	@NotNull
+	@Size(min=2,max=50, message = "Recipe name must be between 2 and 50 characters long.")
+	private String name;
+
+	@NotNull
+	@NotBlank
+	@Size(min=4, max=1000, message = "Directions are required.")
```

Comment:
This is more design consideration, but I think having a minimum size on directions is good, but maybe we should remove the max? I imagine there are recipes that require detailed instructions that might need more than 1000 characters.